clientgui/AsyncRPC.cpp

```diff
@@ -614,7 +638,10 @@ int CMainDocument::RequestRPC(ASYNC_RPC_REQUEST& request, bool hasPriority) {
             // (2) On Signal from RPC thread: locks Mutex again and wakes the main thread.
             m_pRPC_Request_Condition->WaitTimeout(timeToSleep);
 
-            mutexErr = m_pRPC_Request_Mutex->Unlock();
+#if wxDEBUG_LEVEL
+            mutexErr =
+            m_pRPC_Request_Mutex->Unlock();
```

Comment:
I suspect this line shouldn't be in the `#if...#endif` block. Otherwise in release we will never unlock this mutex and will have a deadlock here